acceptance/lib/helper.rb

```diff
@@ -28,7 +28,7 @@ def self.initialize_config(options)
 
     puppet_build_version = get_option_value(options[:puppet_build_version],
                          nil, "Puppet Development Build Version",
-                         "PUPPET_BUILD_VERSION", "3.7.3")
+                         "PUPPET_BUILD_VERSION", "4.0.0")
```

Comment:
In the past we have tried to keep this version string syncronized with the version specified in `ezbake.conf` so in this case it would probably be `0.2.0`